lib/webapi/webhandler.go

```diff
@@ -101,40 +99,29 @@ func NewHandler(cfg WebHandlerConfig) *WebHandler {
 		wh.Handle("GET", route, wh.noLogin(wh.defaultHandler))
 	}
 
-	defaultRoutes := []string{
-		"/web",
-		"/web/",
-		"/web/portal",
-		"/web/portal/*rest",
-		"/web/installer/new/*rest",
-	}
-	for _, route := range defaultRoutes {
-		wh.Handle("GET", route, wh.needsLogin(wh.defaultHandler))
-	}
+	// root
+	wh.Handle("GET", "/web", wh.needsLogin(wh.rootHandler))
+	wh.Handle("GET", "/web/", wh.needsLogin(wh.rootHandler))
 
-	installerRoutes := []string{
-		"/web/installer/site/:site_domain",
-		"/web/installer/site/:site_domain/*rest",
-	}
-	for _, route := range installerRoutes {
-		for _, m := range []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"} {
-			wh.Handle(m, route, wh.needsLogin(wh.installerHandler))
-		}
-	}
+	// portal
+	wh.Handle("GET", "/web/portal", wh.needsLogin(wh.defaultHandler))
+	wh.Handle("GET", "/web/portal/*rest", wh.needsLogin(wh.defaultHandler))
 
-	siteRoutes := []string{
-		"/web/site/:site_domain",
-		"/web/site/:site_domain/*rest",
-	}
-	for _, route := range siteRoutes {
-		for _, m := range []string{"GET", "POST"} {
-			wh.Handle(m, route, wh.needsLogin(wh.siteHandler))
-		}
-	}
+	// installer
+	wh.Handle("GET", "/web/installer/new/*rest", wh.needsLogin(wh.installerHandler))
```

Comment:
What about post, put and other types of requests that were handled before, here and below?

Reply:
there are no need to have them as there is no exposed APIs using these verbs

Reply:
So when bandwagon app does a POST request - how is this handled then?

Reply:
good catch